src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -100,22 +100,27 @@ public function prepare_items() {
 		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $_REQUEST['s'] ) ) {
 			// nonce check if search form submitted.
 			$nonce = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? sanitize_text_field( $_POST['aggregator']['nonce'] ) :  '';
-			if ( wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+			if ( isset( $_GET['s'] ) || wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+				if ( false === filter_var( $_REQUEST['s'], FILTER_VALIDATE_URL ) ) {
+                    $search_term = sanitize_text_field( $_REQUEST['s'] );
+                }else{
```

Comment:
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Expected 1 space after closing brace; 0 found
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after ELSE keyword; 0 found

<sub>hash: 39d03daf8d6b28c30bbaa0ed1ebde03d</sub>